readme.md

```diff
@@ -296,6 +354,11 @@ unzip /tmp/data/berlin.zip -d /tmp/data
 
 # download openstreetmap data
 curl -s https://s3.amazonaws.com/metro-extracts.mapzen.com/berlin_germany.osm.pbf > /tmp/data/berlin.osm.pbf
+
+#download tiger data
```

Comment:
please add to the comment that this is downloading Oregon data since it specifies `41`, it will read more clearly